src/main/java/org/jitsi/videobridge/Channel.java

```diff
@@ -117,6 +117,16 @@
         = new MonotonicAtomicLong();
 
     /**
+     * The time in milliseconds of the last transport related activity to this
```

Comment:
After the latest commits the "transport" activity does not include rtp, but
the doc states that it does. I may be missing something, will check again
when I have a computer.

Boris

On Tuesday, August 16, 2016, George Politis <notifications@github.com>
wrote:

> In src/main/java/org/jitsi/videobridge/Channel.java
> <https://github.com/jitsi/jitsi-videobridge/pull/286#discussion_r74954701>
> :
>
> > @@ -117,6 +117,16 @@
> >          = new MonotonicAtomicLong();
> >
> >      /**
> > +     * The time in milliseconds of the last transport related activity to this
>
> @bgrozev <https://github.com/bgrozev> could you please be a little more
> precise?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jitsi/jitsi-videobridge/pull/286/files/0c7355f740ce425337092252bd598520efb4df33#r74954701>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AER2Y5vtBisLPiFO-1i11pfbaPooEq9Lks5qgdMAgaJpZM4Jk4hN>
> .
>